src/main/java/io/debezium/connector/spanner/task/SyncEventMerger.java

```diff
@@ -117,17 +120,12 @@ public static TaskSyncContext mergeRebalanceAnswer(TaskSyncContext currentContex
                             newMessage.getMessageTimestamp()));
             TaskSyncContext result = builder
                     .build();
-            LOGGER.info("Processed rebalance answer {} from task {} for rebalance generation id {}", newMessage, newMessage.getTaskUid(),
-                    newMessage.getRebalanceGenerationId());
-
-            long newPartitions = result.getNumPartitions() + result.getNumSharedPartitions();
-            if (newPartitions != oldPartitions) {
-                LOGGER.info(
-                        "Task {}, processed rebalance answer {}: {}, task has total partitions {}, num partitions {}, num shared partitions {}, num old partitions {}",
-                        currentContext.getTaskUid(), newMessage, result.getNumPartitions() + result.getNumSharedPartitions(),
-                        result.getNumPartitions(), result.getNumSharedPartitions(), oldPartitions);
+            LOGGER.info(
+                    "Task {}, Processed rebalance answer from task {} for rebalance generation id {}, task has total partitions {}, num partitions {}, num shared partitions {}, num old partitions {}",
+                    currentContext.getTaskUid(), newMessage.getTaskUid(),
+                    newMessage.getRebalanceGenerationId(), result.getNumPartitions() + result.getNumSharedPartitions(),
+                    result.getNumPartitions(), result.getNumSharedPartitions(), oldPartitions);
```

Comment:
I've noticed throughout the code that pre-dates the PR and changes in the PR, where there are uses of `LOGGER.debug` and in other places use `LoggerUtils#debug`.  

I understand the latter is to handle pretty-fying objects for JSON logging; however, to someone who may not be familiar with the code base, it seems odd to have uses of `LOGGER.xxxx` and then have these other wrapper functions.  Perhaps consider unifying this in the long-term?

Reply:
That sounds good! I can open a JIRA later regarding this code cleanup request.